docs/colors.rst

```diff
@@ -88,3 +88,42 @@ Sample: gradients
 .. raw:: html
 
   <img src="colors/gradient_00_expected.png" width=600 class="figure">
+
+Sample: Switch color based on variable
```

Comment:
Fantastic! Can you do the following:

- [ ]  put a working sample in `samples/colors` directory. And then you can include it here with an include in the same way we do in https://github.com/andymeneely/squib/blob/dev/docs/build_groups.rst - with "literalinclude". 
- [ ] name the sample file starting with an underscore

I run the samples directory on my regression testing, so if this ever breaks we'll know. 

I also made some suggestions to use more conventional formatting.

Reply:
Thanks for the hint regarding adding a sample. I added the required files and I think you'll squash the commits on merge, right?

BTW I don't know how to include csv files into the docs like it is used in https://github.com/andymeneely/squib/blob/dev/docs/guides/getting-started/part_1_zero_to_game.rst#data-or-layout. I add a table to the docs with the used data from the sample.